BfA fix

arbi316 opened the pull request:
I've fixed MSBT for BfA 8.0

arbi316 committed:
BfA fix

arbi316 committed:
BfA fix

arbi316 committed:
BfA fix

arbi316 committed:
BfA fix

Toludin reviewed (commented):
Honestly I think you should probably host this as your own update - mine were very small, just a few quick fixes for a friend for the sound file mid-Legion change. I don't use MSBT myself so I think it would make more sense for you to make your own fork, as it's clear you understand this addon better than I do and are more likely to keep it updated moving forward. Nice job!

gmuir committed:
Add missing expected L.MSG_AC and L.MSG_AC_FULL

MSBT would error out at MSBTProfiles.lua lines 1041 and 1047 with string concatenation errors when trying to load. Re-Adding L.MSG_AC and L.MSG_AC_FULL fixes the issue

arbi316 committed:
Merge pull request #2 from gmuir/patch-1

Add missing expected L.MSG_AC and L.MSG_AC_FULL

arbi316 committed:
two strings added and some locale changes

arbi316 commented:
Thanks for your suggestion. I'd maintain this one as possible as I can. But this project should be temporary because the author still maintain his own addons and it will be fixed soon by him.

I'd do my best if you give permission to me even though I'm very noob in github.
